src/BloomBrowserUI/templates/customXMatter/MXBPamphlet-XMatter/MXBPamphlet-XMatter.less

```diff
@@ -25,25 +25,12 @@
 
 .insideBackCover {
     .marginBox {
-        .bloom-editable:only-child {
```

Comment:
We had the same problem (I worked on this with Steve). It appears that these styles were written before we wrapped the inside-back-cover bloom-editable divs in bloom-translationGroup ones, and the rules as written were messing up the ones that control visibility of different languages within a group. We experimented with changing the rules so that they applied to bloom-translationGroups. But (with current xmatter) we can't see that only-child will ever be true for a TG on the inside back cover. And the not-only-child rules didn't make any difference that we could see, so it seemed to us that they had been made obsolete by various improvements in our standard stylesheets and classes.
I suppose we could go back and try to see who introduced those rules, but it seems highly likely that it was you, so if you don't remember what they were for I think we just have to hope that if something is not ending up the way it is wanted someone will let us know, and we can then figure out an appropriate way to further adjust the stylesheet.